.trivyignore

```diff
@@ -1,5 +1,12 @@
-#telegraf MEDIUM
-GHSA-2w8w-qhg4-f78j
+#telegraf
+GHSA-2w8w-qhg4-f78j # MEDIUM
+CVE-2023-2253 # HIGH
```

Comment:
Are you sure we have these vulnerabilities? Why are not the current builds failing then?